main.c

```diff
@@ -87,8 +87,10 @@ int main(int argc, char *argv[]) {
 	if (state.interactive && !(state.options & MRSH_OPT_NOEXEC)) {
 		source_profile(&state);
 	}
-
-	// TODO: set PWD
+	
+	char cwd[PATH_MAX];
+	getcwd(cwd, PATH_MAX);
+	mrsh_env_set(&state, "PWD", cwd, MRSH_VAR_ATTRIB_EXPORT);
```

Comment:
Setting PWD is a little bit complicated: see http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_05_03

You can just implement part of it and leave the rest as a TODO if necessary.

Reply:
Done